typescript/scripting-api/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "wot-typescript-definitions",
```

Comment:
I wonder whether the folder name (currently `scripting-api` should be aligned with the name)

Reply:
@relu91 add "private" to package.json so that it cannot be published

Reply:
Scripting Call 2021-08-09
decided to keep "scripting-api" folder name